### PR TITLE
Possible solution to issue #1

### PR DESCRIPTION
**Pull request by tiagosmartinho**
Hello @sindresorhus!

I saw issues #1 , and I tried to solve it. At first I tried to use the prototype to change the toString () method, but after a few hours, I came up with this solution.

I hope this is what you are looking for with #1.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint new-cap: ["error", { "newIsCap": false }] */
```
Comment (kevva) [inline test.js]: This isn't needed.

File: index.js [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
+
 module.exports = (to, from) => {
 	// TODO: use `Reflect.ownKeys()` when targeting Node.js 6
 	for (const prop of Object.getOwnPropertyNames(from).concat(Object.getOwnPropertySymbols(from))) {
 		Object.defineProperty(to, prop, Object.getOwnPropertyDescriptor(from, prop));
 	}
+
+	const aux = toString(to);
+	to.toString = function () {
+		const comment = `/* ${aux} */ \n	${from}`;
```
Comment (kevva) [inline index.js]:
```js
const comment = `/* ${aux} */\n\t${from}`;
```

File: package.json [hunk 3]
```diff
@@ -39,5 +39,8 @@
   },
   "xo": {
     "esnext": true
+  },
+  "dependencies": {
```
Comment (kevva) [inline package.json]: `dependencies` should be listed before `devDependencies`.

**Comment by kevva** [comment]
Also, it doesn't seem like it does what's mentioned in https://github.com/sindresorhus/mimic-fn/issues/1#issuecomment-255179247.

**Comment by tiagosmartinho** [comment]
Hello @kevva, 

Thank you for the directions.

I thought the output would be something like this: 
```
/* function wrapper () {
     return foo () {};
} */
function foo () {}
```
but now I realize that output should be something like this:

`/* [wrapped with function foo() {}] */`

right?

**Comment by sindresorhus** [comment]
> but now I realize that output should be something like this:
> /* [wrapped with function foo() {}] */

Almost. It should be: `/* [wrapped with foo()] */`

**Comment by sindresorhus** [comment]
Actually, let's just make it:

```
/* Wrapped with foo() */
```

**Comment by sindresorhus** [comment]
Closing for lack of activity. Open a new PR if you're still interested in finishing this.

**Pull request closed without merging**